mbus_packet.cpp

```diff
@@ -261,33 +261,22 @@ uint16_t verifyCrcBytesCmodeB(uint8_t* pByte, uint8_t* pPacket, uint16_t packetS
 {
   uint16_t crc = 0;
   uint16_t i = 0;
-  if (packetSize > 128) {
-    while (i < 126) {
-      crc = crcCalc(crc, pByte[i]);
-      pPacket[i] = pByte[i];
-      ++i;
-    }
-
-    if ((~crc) != (pByte[i] << 8 | pByte[i + 1])) {
+  //Support only for packets with L (lenght byte) < 128 (packetSize < 131)
```

Comment:
why?

Reply:
I wanted to consider here only C1 B frames which consists of first and second block only. (Ignoring frames with additional optional blocks).